charts/collabora-online/0.0.1/test_values.yaml

```diff
@@ -0,0 +1,27 @@
+image:
+  repository: collabora/code
+  tag: 6.4.6.2
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  type: NodePort
+  port:
+    port: 9980
+    nodePort: 30980
+
+env:   
+  domain: nextcloud\.domain\.tld       
+  dictionaries: de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru              
+  username: admin
+  password: changeme
+  extra_params:
+
+securityContext:
+  privileged: false
+
+
```

Comment:
For storage you can copy pasta the storage settings from most test_values.yaml and change the mount/folder/path names accordingly

Reply:
Should i copy both persistence and appvolumemounts?